FuncGameDev/FSharp/Unity/Updater/CameraUpdater.fs

```diff
@@ -0,0 +1,23 @@
+﻿module CameraUpdater
+
+open UnityEngine
+
+let update =
+    // update camera
+    let camera = Camera.main
+    let transform = camera.transform
+
+    let cPosX = transform.position.x |> float
+    let cPosY = transform.position.y |> float
+    let pos = (cPosX, cPosY)
+
+    let cRotX = transform.rotation.x |> float
+    let cRotY = transform.rotation.y |> float
+    let rot = (cRotX, cRotY)
+
+    let cScaleX = transform.localScale.x |> float
+    let cScaleY = transform.localScale.y |> float
+    let scale = (cScaleX, cScaleY)
+
+    CameraBehavior.updateCamera pos, rot, scale, 20.0, 10.0
```

Comment:
Why are you hardcoding width and height to 20? Also, this is not the right place to call the CameraBehavior, though frankly the organization of our gamestate update calls isn't great to begin with. Still: these updaters are for Unity GameObjects, not for the gamestate. These should be separate.